ci-regression.sh

```diff
@@ -22,10 +22,11 @@ cp -p -r "${PWD}" "${TEST_HARNESS_MASTER_DIR}"
 
 pushd "${TEST_HARNESS_MASTER_DIR}"
 
+# stash changes that may have occurred from the install step before checking stuff out
+git stash
```

Comment:
Is this really necessary? Versioned files should be clean on Travis, no? What part of the install would have done this?  I doubt users will appreciate an automated script stashing their changes, so please confirm this is only in the copy of our directory.

Reply:
Like I said above, it was originally "necessary" before I removed the `chmod` calls when I just stored that in git.  I don't know how "necessary" it is now.

Reply:
Then it's fine either way. I wouldn't stress about it. Might be worth logging if this was ever necessary, but I'm not sure how you'd do that, and I'm falling asleep.

Reply:
Ok, so I looked a little closer and found a typo where we set the `GIT_DETACHED` variable. Since I've fixed the only place git checkout complained about changes, I decided to try removing `git stash` for now in de94022593590e98cb508de254f204ae73cd3990.

Reply:
Ok, yeah, I've put the `git stash` commands back where you guys had them: b1fc186c8eb1b5c9ac08aee11567296c6cfc6bf3

That should fix that warning.